lib/manager.php

```diff
@@ -107,6 +107,19 @@ private static function registerDefault()
                 $page == 'modules/actions' && ((($function == 'add' || $function == 'edit') && $save == '1') || $function == 'delete')
             );
         }
+
+        if (rex_addon::exists('yform') && rex_addon::get('yform')->isInstalled() && $addon->getConfig('yform_email')) {
```

Comment:
Der exists-Check ist überflüssig (wenn ein Addon nicht existiert, liefert `rex_addon::get` ein Null-Addon-Objekt).
Statt `isInstalled` besser `isAvailable`.

```suggestion
        if (rex_addon::get('yform')-> isAvailable() && $addon->getConfig('yform_email')) {
```